.changeset/big-bugs-kick.md

```diff
@@ -0,0 +1,7 @@
+---
+'@keystonejs/app-admin-ui': patch
```

Comment:
```suggestion
'@keystonejs/app-admin-ui': minor
```

Since we're exporting some new things, this is an "Addition", aka a "minor" change.